activateEvents.js

```diff
@@ -0,0 +1,76 @@
+// The second IIFE should augment the original one with
+// a function that creates all of the eventHandlers
+// that you need for the application. Name the function
+// activateEvents.
+
+var CarLot = (function(carlotCars) {
+
+  //create a reference to the cards of the cars
+  var cars = document.getElementsByClassName('cards');
+
+  //a reference to the user input text box
+  var userInput = document.getElementById('user-input');
+
+
+  var inputChange = "";
+
+  //create a function to add all event listeners to elements
+  carlotCars.activateEvents = function() {
+
+    //this loop adds event listeners to each card
+    for ( let i = 0; i < cars.length; i++) {
+      cars[i].addEventListener('click', function() {
+        let color = "lightgray";
+        carlotCars.highlightCard(cars[i], color);
+        });
+      };
+  },//end carlotCars.activateEvents method
+
+  carlotCars.getCarsFromDom = function() {
+    return cars;
+  },//end carlotCars.getCarFromDom function
+
+
+  carlotCars.userInputListener = function() {
+
+    userInput.addEventListener('click', carlotCars.expandBox);
+
+  },//end carlotCars.userInputListener function
+
+
+  //create a function to grab description tag from a selected element
+  carlotCars.selectDescription = function() {
+
```

Comment:
It looks like you need to add you focus() for your user-input id in this function.